mathesar/views.py

```diff
@@ -107,14 +107,15 @@ def render_schema(request, database, schema):
 
 def home(request):
     database = get_current_database(request, None)
-    schema = get_current_schema(request, None, database)
```

Comment:
Since there will be no schema to be rendered, hence removed this extra query. 
NOTE: Preloading has also been removed.

Reply:
What is meant by preloading removed in this context?